lib/Conversion/TritonGPUToLLVM/TritonGPUToLLVM.cpp

```diff
@@ -3573,8 +3577,29 @@ DotOpConversion::convertMMA884(triton::DotOp op, DotOpAdaptor adaptor,
         (m * 2 + 1) + (n * 4 + 2) * numM, // row3
         (m * 2 + 1) + (n * 4 + 3) * numM,
     }};
+    return idx;
+  };
+
+  { // convert the acc's value from accumuator-external layout to
+    // accumulator-internal layout.
+    SmallVector<Value> acc2(acc);
```

Comment:
1, will it be better to name it as something like "accInit"?
2, it's not necessary to initialize acc2 with acc, just "SmallVector<Value> acc2(resSize);" should be good enough.

Reply:
done